src/com/theironyard/novauc/ReadWriteJson.java

```diff
@@ -0,0 +1,102 @@
+package com.theironyard.novauc;
+
+import jodd.json.JsonParser;
+import jodd.json.JsonSerializer;
+
+import java.io.Console;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+import static com.theironyard.novauc.Main.scanner;
+import static com.theironyard.novauc.Main.scannerInt;
+
+
+/**
+ * Created by octavio on 2/17/17.
+ */
+
+
+public class ReadWriteJson {
+
+    public static Scanner userInput = new Scanner(System.in);
+
+
+    public static void main(String[] args) throws IOException {
+
+
+
+        Survey z = new Survey();
+        z.brand = "Hyundai";
```

Comment:
Nice job Octavio.  Instead of using a constructor with no parameters here and setting each value one by one.  You could use a constructor that takes 5 parameters.  Example Survey z = new Survey ("hyundai", "sonata", 2016, "White", 10);  You would then have to make a constructor in your Survey class to match.  The code would be a little bit cleaner that way.  Overall nice job.